PowerShellTools/PowerShellToolsPackage.cs

```diff
@@ -130,6 +134,14 @@ public override Type GetLibraryManagerType()
             return null;
         }
 
+        public static bool UseOutProc

```

Comment:
What if we're on a 32bit operating system? It makes very little sense to still go out of proc though we can.